multiple phantomjs options as an array

yubowenok opened the pull request:
When multiple phantomjs options are passed in, e.g.
```js
gulp.src('index.html')
    .pipe(qunit({
      'phantomjs-options': ['--ignore-ssl-errors=true', '--web-security=false']
    })
```
The current code will push the array into childArgs, which will result in childProcess.execFile failing to execute phantomjs properly. It will give "Invalid values for 'ignore-ssl-errors' option." It seems that each array element in the 2nd argument of childProcess.execFile must contain a single flag. Passing in the concatenated string '--ignore-ssl-errors=true --web-security=false' does not work.

jonkemp commented:
Can you provide more detail about what this patch is for? Is there a failing test that we can add?

yubowenok commented:
Added some details in the PR description. Added a failing test.
![image](https://cloud.githubusercontent.com/assets/1314429/14856318/94d67ab6-0c65-11e6-9d62-7bf5dede5b6e.png)

jonkemp commented on index.js:
```diff
@@ -19,7 +19,11 @@ module.exports = function (params) {
 
         if (options['phantomjs-options'] && options['phantomjs-options'].length) {
-            childArgs.push(options['phantomjs-options']);
+            if (typeof options['phantomjs-options'] === 'string') {
```
Instead of checking for a string, could we check to see whether it is an array or not? Example,
```js
if (!Array.isArray(options['phantomjs-options'])) {}
```

yubowenok commented:
Used Array.isArray.

jonkemp commented:
Ok. Can you please squash commits?

yubowenok commented:
Done
